cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -6,9 +6,9 @@ import (
 
 	"github.com/alecthomas/kong"
 	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/square/spiffe-aws-assume-role/cmd/spiffe-aws-assume-role/cli/mappers"
 	"github.com/square/spiffe-aws-assume-role/pkg/credentials"
 	"github.com/square/spiffe-aws-assume-role/pkg/processcreds"
-	"github.com/square/spiffe-aws-assume-role/pkg/util"
 )
```

Comment:
The Mapper interface is tied to Kong so it's definitely more of a CLI thing than anything else (meaning, I don't think it belongs in pkg but maybe I'm just wrong). However, since it's not exactly specific to this particular command I could see moving it up a couple levels to cmd/mappers.